src/main/java/beam/analysis/physsim/PhyssimCalcLinkStats.java

```diff
@@ -52,7 +52,7 @@
     private BeamConfig beamConfig;
     private Network network;
     private OutputDirectoryHierarchy controllerIO;
-    private BeamCalcLinkStats linkStats;
+    public BeamCalcLinkStats linkStats;
```

Comment:
Recommended suggestion:
- Don´t expose fields as public. It breaks encapsulation. Let it private and if needed create accessor/mutator method.